jf_agent/__init__.py

```diff
@@ -48,6 +48,15 @@ def write_file(outdir, filename_prefix, compress, results):
         with open(f'{outdir}/{filename_prefix}.json', 'w') as outfile:
             outfile.write(json.dumps(results, indent=2, default=str))
 
+def write_status_file(outdir, type, status):
+    with open(f'{outdir}/status.json', 'r') as status_file:
+        status_json = json.load(status_file)
+        body = {
+            'type': type,
+            'status': status
+        }
+        status_json.append(body)
```

Comment:
just want to make sure we've tested the cases where you are appending to an empty file and also appending to a non-empty file

Reply:
So the first `write_file()` call I added that puts in a `[]` to the status file makes it such that these append calls are never on an empty file.